EightfoldAI/octuple
# Pill component readOnly prop

## SUMMARY:
Adding readOnly prop to pills component, default it to false
## GITHUB ISSUE (Open Source Contributors)

## JIRA TASK (Eightfold Employees Only):
https://eightfoldai.atlassian.net/jira/software/c/projects/ENG/boards/50?modal=detail&selectedIssue=ENG-30670
## CHANGE TYPE:


## TEST COVERAGE:


## TEST PLAN:
- go to storybook's Pill component canvas
- switch readOnly prop to true
- check that the pills are not clickable anymore
